tech_owner.rst

```diff
@@ -0,0 +1,94 @@
+Tech owner
+==========
+
+Bij nens is veel vrijheid om met nieuwe technologieen te experimenteren en die in productie te nemen. Dat maakt ons flexibel maar ook kwetsbaar als we niet de verantwoordelijkheid oppakken die bij die vrijheid hoort.
+
+Meestel begint het experimenteren met nieuwe technologie op basis van interesse of skills van 1 persoon. Maar zo gauw die technologie rijp is voor productie is het van belang de kennis van de technologie in de organisatie te borgen.
```

Comment:
Ik zou er ook een financiële paragraaf aan toevoegen. Zoiets als dit?

_Belangrijk is ook het budget in de gaten te houden. Iedere extra service ("laten we sentry gebruiken") is een website die op een server staat die elke twee jaar verplaatst moet worden bij een OS update. En die up-to-date gehouden moet worden. Daar moet tijd & budget voor vrijgemaakt worden. Ook het ècht in productie nemen van iets betekent vaak werk. Daar moet budget voor geregeld worden._ 

Ik heb in december best wat opschoontijd besteed aan wat van onze infra, maar de SSO draait nog niet op 14.04. Sentry is nog niet geüpdated. Trac is wel over (en up to date). Packages nog op 12.04. Enz. Moet dan logstash er nog bijkomen, bijvoorbeeld? Is daar 3 weken budget voor om het echt netjes op te zetten en in meerdere producten te integreren?